packages/seo/metadata.ts

```diff
@@ -45,6 +47,8 @@ export const createMetadata = ({
     },
     publisher,
     twitter: {
+      title: parsedTitle,
```

Comment:
I was under the impression that `metadata.twitter.title` and `.description` inherit from the parent?

Reply:
That might be the case. Ill close the pr then!